content/id/docs/setup/best-practices/cluster-large.md

```diff
@@ -0,0 +1,88 @@
+---
+reviewers:
+- davidopp
+- lavalamp
+title: Pertimbangan untuk klaster besar
+weight: 10
+---
+
+Klaster adalah kumpulan simpul (*{{< glossary_tooltip text="nodes" term_id="node" >}}*) berupa mesin fisik ataupun virtual yang menjalankan agen Kubernetes, dan dikelola oleh {{< glossary_tooltip text="panel kontrol" term_id="control-plane" >}}. Kubernetes {{< param "version" >}} mendukung klaster hingga 5000 simpul. Lebih tepatnya, Kubernetes didesain untuk mengakomodasi konfigurasi yang mendukung kriteria dibawah ini:
```

Comment:
I think we need to keep "node" as it is because "simpul" has different meaning in the KBBI.
https://kbbi.kemdikbud.go.id/entri/simpul

Also, "control-plane" must be "bidang-kontrol"